src/SelectMenu/SelectMenuModal.tsx

```diff
@@ -99,10 +99,10 @@ const ModalWrapper = styled.div<StyledModalWrapperProps>`
 type SelectMenuModalInternalProps = Pick<StyledModalProps, 'width'> & ComponentProps<typeof ModalWrapper>
 
 const SelectMenuModal = React.forwardRef<HTMLDivElement, SelectMenuModalInternalProps>(
-  ({children, theme, width, ...rest}, forwardedRef) => {
```

Comment:
I'm not sure how valuable the `no-shadow` rule is. I think it's usually fine to shadow variables 🤷‍♂️ Unless anyone has strong opinions otherwisel, I'd say we can disable that rule in the config

Reply:
while you _can_ shadow variables, I think it's safer to avoid it.  Definitely easy to lose track and think you are using the wrong variable if it is shadowed.  That said, I'd be happy to change the way that I fixed these particular instances, for example we could change the name of the import rather than changing the name of the spread prop.

Reply:
I'm fine leaving this as-is 👍